### PR TITLE
Added Ruby Style Guide

### PR DESCRIPTION
**Pull request by robot0**
Description

Why?

To provide IT Glue's Ruby Style Guide based on our best practices.

What's Changing?

Adding Ruby Styles guide to our codestyle guides.

### CODE REVIEW

**Review by aglumova — APPROVED** [review]

File: ruby/README.md [hunk 1]
```diff
@@ -0,0 +1,1750 @@
...
+    def is_eligible?(user)
+      Trebuchet.current.launch?(ProgramEligibilityHelper::PROGRAM_TREBUCHET_FLAG) &&
+        is_in_program?(user) &&
+        program_not_expired
+    end
+    ```
+
+### Inline
+
+* Never leave trailing whitespace.
```
Comment (aglumova) [inline ruby/README.md:128]: Could you please prepare `github` action to push this code style as lib to the ruby dependency manager and please add info about how to add it to the any Ruby project dependencies and use it in the code style.

**Review by aglumova — CHANGES_REQUESTED** [review]

**Comment by aglumova** [comment]
Please add the next info the the PR description

```
Description

Why?

[Brief description of why this PR is necessary.]

What's Changing?

[Brief description of what this PR does.]
```

This is the common template for all and it's required.

File: ruby/README.md (hunk 1 above)
Comment (robot0) [reply ruby/README.md:128]: Added github action

**Review by aglumova — APPROVED** [review]

**Comment by aglumova** [comment]
Happy to merge!

**Comment by robot0** [comment]
Yay!

**Pull request merged**